### ISSUE TITLE
Broken shorthand order when using source maps

### ISSUE THREAD

**Issue by radium-v** [jakubpawlowicz/clean-css#743]
Hi,

I noticed this squishy bug today. I'm not sure what's causing it, but I've been able to reproduce it.

Input:
```css
.box {
  background: 50% 50% / 100% auto no-repeat currentColor;
}
```

Expected Output:
```css
.box{background:50% 50%/100% auto no-repeat currentColor}
```
If I don't use a source map, the output above is returned. If I do use a source map, the output is messed with slightly:
```bash
$ cleancss --source-map -o css-bundle/box.min.css css/box.css
```
```css
.box{background:50% 50% auto/100% no-repeat currentColor}/*# sourceMappingURL=box.min.css.map */
```
```json
{"version":3,"sources":["css/box.css"],"names":[],"mappings":"AAAA,iBACC,WAAY,IAAI,IAAU,KAAL,KAAU,UAAU"}
```

The `background-size` and `background-position` values get mixed up, changing from `50% 50% / 100% auto` to `50% 50% auto / 100%`. If I change the input and remove some spaces, it works (`50% 50%/100% auto`).

TL;DR if you use `background-size` in the shorthand `background` property, the `/` can't have whitespace around it.

**Comment by jakubpawlowicz** [comment]
I can see it too. Fix is coming!

**Comment by jakubpawlowicz** [comment]
Fixed in 3.4.11.
